bigframes/ml/llm.py

```diff
@@ -892,8 +892,12 @@ def fit(
         Returns:
             GeminiTextGenerator: Fitted estimator.
         """
-        if self._bqml_model.model_name.startswith("gemini-1.5"):
-            raise NotImplementedError("Fit is not supported for gemini-1.5 model.")
+        # Support gemini-1.5 and gemini-pro
+        supported_models = ["gemini-pro", "gemini-1.5-pro-002", "gemini-1.5-flash-002"]
+        if self.model_name not in supported_models:
+            raise NotImplementedError(
+                "Score is not supported models other than gemini-pro or gemini-1.5 model."
```

Comment:
be more explicit to "gemini-1.5-pro-002" and "gemini-1.5-flash-002".

Since other gemini 1.5 endpoints aren't supported.

Reply:
done